app/helpers/application_helper.rb

```diff
@@ -8,4 +8,8 @@ def pagination_info(object)
   def display(error)
     content_tag(:span, error.join(','), class: 'errors') if error.present?
   end
+
+  def company_name
+  	Company.where(subdomain: request.subdomain).first.name.capitalize
+  end
```

Comment:
what if `Company.where(subdomain: request.subdomain).first` is nil?
maintain consistency, either use `find_by()` or `where().first`